templates/pulp.conf.erb

```diff
@@ -17,15 +17,19 @@ SSLInsecureRenegotiation on
 
 WSGIProcessGroup pulp
 WSGIApplicationGroup pulp
-WSGIDaemonProcess pulp user=apache group=apache processes=1 threads=8 display-name=%{GROUP}
+WSGIDaemonProcess pulp user=apache group=apache processes=3 display-name=%{GROUP}
 
 # DEBUG - uncomment the next 2 lines to enable debugging
 #WSGIRestrictStdin Off
 #WSGIRestrictStdout Off
 
 WSGISocketPrefix run/wsgi
-WSGIScriptAlias /pulp/api /srv/pulp/webservices.wsgi
-WSGIImportScript /srv/pulp/webservices.wsgi process-group=pulp application-group=pulp
+WSGIScriptAlias /pulp/api /usr/share/pulp/wsgi/webservices.wsgi
+WSGIImportScript /usr/share/pulp/wsgi/webservices.wsgi process-group=pulp application-group=pulp
+
+<Directory /usr/share/pulp/wsgi>
+    Require all granted
```

Comment:
I think you need to account for differences in apache versions: 

https://github.com/Katello/puppet-katello/blob/master/templates/etc/httpd/conf.d/05-foreman.d/katello.conf.erb#L11-L15